e2e-testing/cypress/functions/DataViewFunctions/DataViewFunctions.js

```diff
@@ -25,6 +25,32 @@ class DataViewFunctions {
       .children(DataViewLocators.postItem)
       .contains('Automated Title Response');
   }
+  search_and_verify_results(keyword) {
+    this.click_data_view_btn();
```

Comment:
Before searching for the keyword, could you add a post with the keyword so that  the post exists that you are searching for.

Reply:
do you mean to create a new post in the test block? because there are posts with the election keyword before now...

Reply:
An update has been made for this, kindly review @shakirandagire